src/System.Runtime.CompilerServices.Unsafe/Directory.Build.props

```diff
@@ -3,6 +3,7 @@
   <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props, $(MSBuildThisFileDirectory)..))" />
 
   <PropertyGroup>
+    <IsPackable>true</IsPackable>

```

Comment:
I'm aware this would also affect the tests csproj but keep in mind we pass /p:SkipTests=true in the CI commands and the official build commands.

Reply:
Will this cause the https://github.com/dotnet/maintenance-packages/blob/f7fd7fadbd205e9751ea76a6a89119f30c2e68dc/src/System.Runtime.CompilerServices.Unsafe/ver/Versioning.csproj to be packed too?

Reply:
That didn't seem to be the case:

```
ls .\artifacts\packages\Release\Shipping\

        Directory: C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
-a---        2024-10-11  10:11 AM          31631   Microsoft.Bcl.HashCode.6.0.0-ci.24511.1.nupkg
-a---        2024-10-11  10:11 AM          57606   Microsoft.Bcl.HashCode.6.0.0-ci.24511.1.symbols.nupkg
-a---        2024-10-11  10:11 AM          81793   Microsoft.IO.Redist.6.1.0-ci.24511.1.nupkg
-a---        2024-10-11  10:11 AM         109826   Microsoft.IO.Redist.6.1.0-ci.24511.1.symbols.nupkg
-a---        2024-10-11  10:11 AM          20809   System.Buffers.4.6.0-ci.24511.1.nupkg
-a---        2024-10-11  10:11 AM          32076   System.Buffers.4.6.0-ci.24511.1.symbols.nupkg
-a---        2024-10-11  10:11 AM          46838   System.Json.4.8.0-ci.24511.1.nupkg
-a---        2024-10-11  10:11 AM          75532   System.Json.4.8.0-ci.24511.1.symbols.nupkg
-a---        2024-10-11  10:11 AM         116313   System.Memory.4.6.0-ci.24511.1.nupkg
-a---        2024-10-11  10:11 AM         185248   System.Memory.4.6.0-ci.24511.1.symbols.nupkg
-a---        2024-10-11  10:11 AM          72737   System.Net.WebSockets.WebSocketProtocol.5.1.0-ci.24511.1.nupkg
-a---        2024-10-11  10:11 AM         107354   System.Net.WebSockets.WebSocketProtocol.5.1.0-ci.24511.1.symbols.nupkg
-a---        2024-10-11  10:11 AM          69536   System.Numerics.Vectors.4.6.0-ci.24511.1.nupkg
-a---        2024-10-11  10:11 AM         103922   System.Numerics.Vectors.4.6.0-ci.24511.1.symbols.nupkg
-a---        2024-10-11  10:11 AM          30119   System.Reflection.DispatchProxy.4.8.0-ci.24511.1.nupkg
-a---        2024-10-11  10:11 AM          49664   System.Reflection.DispatchProxy.4.8.0-ci.24511.1.symbols.nupkg
-a---        2024-10-11  10:11 AM          20730   System.Runtime.CompilerServices.Unsafe.6.1.0-ci.24511.1.nupkg
-a---        2024-10-11  10:11 AM          22612   System.Runtime.CompilerServices.Unsafe.6.1.0-ci.24511.1.symbols.nupkg
-a---        2024-10-11  10:11 AM          24400   System.Threading.Tasks.Extensions.4.6.0-ci.24511.1.nupkg
-a---        2024-10-11  10:11 AM          36876   System.Threading.Tasks.Extensions.4.6.0-ci.24511.1.symbols.nupkg
-a---        2024-10-11  10:11 AM          17998   System.Xml.XPath.XmlDocument.4.4.0-ci.24511.1.nupkg
-a---        2024-10-11  10:11 AM          32596   System.Xml.XPath.XmlDocument.4.4.0-ci.24511.1.symbols.nupkg
```

Reply:
I see, I guess since it's not included in the root build.proj it doesn't have pack invoked?

I did notice that the tests are still packed in the PR build: https://dev.azure.com/dnceng-public/public/_build/results?buildId=840147&view=logs&j=ec685536-f43b-5a9a-92ff-89175ccc3932&t=1f5934e1-94d3-500b-1b02-b1064dab718f&l=149

Did you consider any different way to share the version without causing this problem?  Maybe instead of a d.b.p that gets imported by the tests it could be a version.props that's imported by src and ver?

Reply:
I can explore that. I opened an issue to address this separately so I can get unblocked on this. https://github.com/dotnet/maintenance-packages/issues/131